### PR TITLE
Update README and process for 0.5 tag [skip ci]

### PR DESCRIPTION
**Pull request by hickeng**
Updates the README for the 0.5 tag and updates the release process so that we do this in consistent fashion for future tags.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,17 +11,17 @@ See [VIC Containers Architecture](doc/design/arch/arch.md) for a high level over
...
-* experimental vCenter support
-* early volume support [--volume-store](doc/user/usage.md#configuring-volumes-in-a-virtual-container-host)
-* direct mapping of vSphere networks [--container-network](doc/user/usage.md#exposing-vsphere-networks-within-a-virtual-container-host)
-* port forwarding on bridge network
+* basic function for most of the core lifecycle operations: pull, create, start, attach, run, stop, rm, logs.
+* early vCenter support, leveraging DRS for initial placement.
+* volume support [--volume-store](doc/user/usage.md#configuring-volumes-in-a-virtual-container-host) - SIOC is not integrated but can be set as normal.
+* direct mapping of vSphere networks [--container-network](doc/user/usage.md#exposing-vsphere-networks-within-a-virtual-container-host) - NIOC is not integrated but can be set as normal.
+* dual-mode management - IP addresses are reported as normal via vSphere UI, guest shutdown via the UI will trigger delivery of container STOPSIGNAL, restart will relaunch container process.
+* logs command - note that follow (`-f`) is not yet implemented
```
Comment (mdubya66) [inline README.md]: Do we want to mention vicadmin?

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

**Comment by mdubya66** [comment]
lgtm

**Comment by hickeng** [comment]
poke

**Pull request merged**